docs/source/get_started/modular_design.rst

```diff
@@ -28,9 +28,21 @@ This class can contain learnable parameters and `NaN` (missing value) handling.
 It takes :class:`~torch_frame.data.TensorFrame` as input and applies stype-specific feature encoder (specified via :obj:`stype_encoder_dict`) to :obj:`~torch.Tensor` of each stype to get embeddings for each :obj:`~torch_frame.stype`.
 The embeddings of different :obj:`stypes<torch_frame.stype>` are then concatenated to give the final 3-dimensional :obj:`~torch.Tensor` :obj:`x` of shape :obj:`[batch_size, num_cols, channels]`.
 
+.. note::
+    There are two types of :obj:`stypes<torch_frame.stype>`--user-facing and internal.
+    User facing :obj:`stypes<torch_frame.stype>` is declared on the :class:`~torch_frame.data.Dataset` level, where users can specify the :class:`~torch_frame.stype` for each column.
+    The raw data of the user-facing :obj:`stype<torch_frame.stype>` will be converted into data of internal :obj:`stype<torch_frame.stype>` during materialization.
```

Comment:
Can we make the following clear throughout?

The raw data is in the format of `DataFrame`
data of internal is in the format of PyTorch `Tensor` or `TensorFrame`.

Reply:
I meant raw data of the column.